Xamarin.MacDev/MacOSXSdk.cs

```diff
@@ -32,15 +32,15 @@ namespace Xamarin.MacDev
 {
 	public class MacOSXSdk : IAppleSdk
 	{
-		List<MacOSXSdkVersion> knownOSVersions = new List<MacOSXSdkVersion> {
-			MacOSXSdkVersion.V10_7,
-			MacOSXSdkVersion.V10_8,
-			MacOSXSdkVersion.V10_9,
-			MacOSXSdkVersion.V10_10,
-			MacOSXSdkVersion.V10_11,
-			MacOSXSdkVersion.V10_12,
-			MacOSXSdkVersion.V10_13,
-            		MacOSXSdkVersion.V10_14
+		List<AppleSdkVersion> knownOSVersions = new List<AppleSdkVersion> {
+			AppleSdkVersion.V10_7,
+			AppleSdkVersion.V10_8,
+			AppleSdkVersion.V10_9,
+			AppleSdkVersion.V10_10,
+			AppleSdkVersion.V10_11,
+			AppleSdkVersion.V10_12,
+			AppleSdkVersion.V10_13,
+			AppleSdkVersion.V10_14,
```

Comment:
not sure who use the API but should include 10.15, 11.[0-3] ?
Also not so clear about the new minor version of 11.x

Reply:
I believe VSMac are using these.

I didn't want to remove any that VSMac might be using (nor add any VSMac isn't using).